Add google cloud (#18)
NeowayLabs/packer-images: README.md

$ make setup
```

## Select your cloud provider

At moment, this project supports:
- [Azure](docs/azure.md)
- [Google Cloud](docs/gcp.md)
- [Digital Ocean](docs/do.md)

Feel free to add more builders. The list of available builders are [here](https://www.packer.io/docs/builders/index.html).

###

## Arguments

> On this project we separate the multi cloud schema and tools, with arguments on execution of Makefile. The valid arguments are:

Reviewer: ```suggestion
On this project we separate the multi cloud schema and tools using arguments when executing make. The valid arguments are:
```
